spec/features/reward_filter_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+# frozen_string_literal: true
+RSpec.describe 'Reward filter spec', type: :feature do
+  let!(:employee) { create(:employee) }
+  let!(:category1) { create(:category, title: 'Category 1') }
+  let!(:category2) { create(:category, title: 'Category 2') }
+  let!(:reward1) { create(:reward, category: category1) }
+  let!(:reward2) { create(:reward, category: category2) }
+
+  before do
+    login_as employee, scope: :employee
+    visit root_path
+  end
+
+  it 'tests filtering rewards' do
+    click_link 'Rewards'
+    click_link category1.title
+    expect(page).to have_content reward1.title
+    expect(page).not_to have_content reward2.title
```

Comment:
I believe  you could use `expect(page).to have_no_content`  so it's more readable.

Reply:
Also, as far as I remember `have_no_content` is more secure, because it waits until the page is fully loaded. While `not_to` doesn't performs the check immediately, which can result in false positive

Reply:
For me, both are readable, but it's good to know that there are different opinions (because it should be clear and legible not only for me 🙂). I decided to change it mainly to avoid false positives.